docs/pages/develop/development-builds/create-a-build.mdx

```diff
@@ -14,17 +14,47 @@ import { BookOpen02Icon } from '@expo/styleguide-icons';
 
 Development builds can be created with [EAS Build](/build/introduction/) or [locally](/guides/local-app-development/) on your computer if you have Android Studio and Xcode installed.
 
-<Video url="https://www.youtube.com/embed/LUFHXsBcW6w" />
+In this guide, you'll find information on creating a development build with EAS and installing it on an emulator/simulator or a physical device to continue developing your app.
+
+> If you are looking to create a development build **locally**, rather than remotely on EAS, you can [create local builds with `npx expo run:[android|ios]`](/guides/local-app-development/#local-builds-with-expo-dev-client) or [with `eas build --local`](/build-reference/local-builds/).
```

Comment:
```suggestion
> If you are looking to create a development build **locally**, rather than remotely on EAS, you can [create local builds with `npx expo run:[android|ios]`](/guides/local-app-development/#local-builds-with-expo-dev-client) or with [`eas build --local`](/build-reference/local-builds/).
```